includes/polling/mempools/cemp.inc.php

```diff
@@ -2,12 +2,21 @@
 
 $oid = $mempool['mempool_index'];
 
+$HCoids = array(
+    "cempMemPoolHCUsed.$oid",
+    "cempMemPoolHCFree.$oid",
+    "cempMemPoolHCLargestFree.$oid",
+);
 $oids = array(
     "cempMemPoolUsed.$oid",
     "cempMemPoolFree.$oid",
     "cempMemPoolLargestFree.$oid",
 );
-$data = snmp_get_multi_oid($device, $oids, '-OUQ', 'CISCO-ENHANCED-MEMPOOL-MIB');
-
+$HCdata = snmp_get_multi_oid($device, $HCoids, '-OUQ', 'CISCO-ENHANCED-MEMPOOL-MIB');
```

Comment:
You could probably just use `$data` here then change the if check to be inverse.